modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/output/CsvFormatter.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.ballerina.messaging.broker.client.output;
+
+import io.ballerina.messaging.broker.client.resources.Binding;
+import io.ballerina.messaging.broker.client.resources.Consumer;
+import io.ballerina.messaging.broker.client.resources.Exchange;
+import io.ballerina.messaging.broker.client.resources.Queue;
+
+/**
+ * Print backend responses in csv format.
+ */
+public class CsvFormatter implements ResponseFormatter {
+
+    /**
+     * Name of this formatter class. This will be used when displaying help logs.
+     */
+    private static final String FORMATTER_NAME = "csv";
+
+    @Override
+    public void printExchanges(Exchange[] exchanges) {
+        if (exchanges.length == 0) {
+            return;
+        }
+        String printTemplate = "%s,%s,%s%n";
+        OUT_STREAM.printf(printTemplate, Exchange.NAME, Exchange.TYPE, Exchange.DURABLE);
+        for (Exchange exchange : exchanges) {
+            OUT_STREAM.printf(printTemplate.replaceFirst("%s", "\"%s\""), exchange.getName(), exchange.getType(),
+                    String.valueOf(exchange.isDurable()));
+        }
+    }
+
+    @Override
+    public void printQueues(Queue[] queues) {
+        if (queues.length == 0) {
+            return;
+        }
+        String printTemplate = "%s,%s,%s,%s,%s,%s%n";
+        OUT_STREAM.printf(printTemplate, Queue.NAME, Queue.CONSUMER_COUNT, Queue.CAPACITY, Queue.SIZE, Queue.DURABLE,
+                Queue.AUTO_DELETE);
+        for (Queue queue : queues) {
+            OUT_STREAM.printf(printTemplate.replaceFirst("%s", "\"%s\""), queue.getName(),
```

Comment:
Maybe we should define a constant for `"\"%s\""` since it is used in 3 places.

Reply:
+1